src/components/Task/Task.js

```diff
@@ -73,32 +37,25 @@ const Task = ({
                 >
                     <BsTrashFill />
                 </div>
-                {
-                    !noRightArrow &&
-                    <div 
-                        className="icon"
-                        onClick={() => changeTaskStage('right')}
-                    >
-                        <BsArrowRight />
-                    </div>
-                }
             </div>
             {isDeleteModalOpened &&
                 <DeleteTaskModal
                     setIsDeleteModalOpened={setIsDeleteModalOpened}
                     isDeleteModalOpened={isDeleteModalOpened}
-                    setAllTasks={setAllTasks}
-                    allTasks={allTasks}
                     _id={_id}
                 />
             }
             {isUpdateModalOpened &&
                 <UpdateTaskModal
                     setIsUpdateModalOpened={setIsUpdateModalOpened}
                     isUpdateModalOpened={isUpdateModalOpened}
-                    setAllTasks={setAllTasks}
-                    setTaskText={setTaskText}
-                    allTasks={allTasks}
+                    task={task}
+                />
+            }
+            {isTaskInfoModalOpened &&
```

Comment:
Same situation, {isTaskInfoModalOpened} if it would be false component don't render & why we need to pass it to component?